redbase/implementation/redbase-sm.md

```
@@ -0,0 +1,124 @@
+# The System Management Component RedBase Part 3: SM
+
+Source: <https://web.stanford.edu/class/cs346/2015/redbase-sm.html>
+
+Due Sunday, May 10, 11:59PM
+
+- [Introduction](#intro)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)